rd/inverse.h

```diff
@@ -0,0 +1,14 @@
+//
```

Comment:
inverse.h and base.h are created rather than using all_funcitons.h, because all_functions.h includes triangle2D.h, and in triangle2D.h mat_inv, max_val, min_val are called. We shouldn't include all_functions.h in the triangle class, but include inverse.h and base.h individually instead.